server/controllers/uncategorised.js

```diff
@@ -248,7 +276,9 @@ exports.listInExAccounts = function (req, res, next) {
 
   function process(accounts) {
```

Comment:
Are 1_2 accounts really income/exchange accounts?  I thought they were capital accounts.  Why are they in the filter for `exports.listInExAccounts`?